### PR TITLE
docs: add Jira docs

### PR DESCRIPTION
**Pull request by jakiuncle**
Fix: https://github.com/casdoor/casdoor/issues/1255

### CODE REVIEW

**Comment by Selflocking** [comment]
image

File: docs/integration/java/jira oidc.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+## Step1. Deploy Casdoor and Jira
+
+Firstly, the [Casdoor](/docs/basic/server-installation) and Jira should be deployed.
+
+After a successful deployment, you need to ensure:
+
+1. Set Jira URL(Plans -> Administration -> System -> General configuration) to `Jira_HOSTNAME`.
+![Jira URL](/img/integration/java/jira-oidc/Jira_HOSTNAME.png)
+2. Casdoor can be logged in and used normally.
+3. My use default pord so my CASDOOR_HOSTNAME = 'http://localhost:8000'
```
Comment (leo220yuyaodog) [inline docs/integration/java/jira oidc.md:24]:
```suggestion
3. CASDOOR_HOSTNAME = 'http://localhost:8000'. When deploy Casdoor in `prod` mode. See [producation mode](https://casdoor.org/docs/basic/server-installation#production-mode).
```

**Review by leo220yuyaodog — CHANGES_REQUESTED** [review]

File: docs/integration/java/jira oidc.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Open your favorite browser and visit: **http://`CASDOOR_HOSTNAME`/.well-known/openid-configuration**, you will see the OIDC configure of Casdoor.
+
+## Step3. Configure Jira
+
+1. You should install a app to support OAuth ![Jira App](/img/integration/java/jira-oidc/Jira_install.png)
+2. You should config this app ![Jira Config](/img/integration/java/jira-oidc/Jira_Config.png)
+   - Application you should select Custom OpenId
+   - You can find Client Id and Client Secret in Casdoor application page.
+     - `Token server url`: **http://`CASDOOR_HOSTNAME`/api/login/oauth/access_token**
```
Comment (leo220yuyaodog) [inline docs/integration/java/jira oidc.md:43]: Too much indentation. It's not very nice here.

image

**Comment by Selflocking** [comment]
One more thing: change your pr name to `docs: add Jira docs`.
https://casdoor.org/docs/contributing#semantic-prs

File: sidebars.js [hunk 3]
```diff
@@ -284,6 +284,7 @@ module.exports = {
             },
             "integration/java/jenkins plugin",
             "integration/java/jenkins oidc",
+            "integration/java/jira oidc",
```
[the remark this answers is not in the document]
Comment (hsluoyz) [reply sidebars.js:287]: @jakiuncle use `jira` directly as name

**Review by Selflocking — APPROVED** [review]

File: docs/integration/java/jira.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## Step1. Deploy Casdoor and Jira
+
+Firstly, the [Casdoor](/docs/basic/server-installation) and [Jira](https://www.atlassian.com/software/jira/guides/getting-started/overview) should be deployed.
+
+After a successful deployment, you need to ensure:
+
+1. Set Jira URL(Plans -> Administration -> System -> General configuration) to `Jira_HOSTNAME`.
+![Jira URL](/img/integration/java/jira/Jira_HOSTNAME.png)
+2. Casdoor can be logged in and used normally.
+3. I use default prod so my CASDOOR_HOSTNAME = 'http://localhost:8000'.When deploy Casdoor in `prod` mode. See [production mode](https://casdoor.org/docs/basic/server-installation#production-mode).
```
Comment (leo220yuyaodog) [inline docs/integration/java/jira.md:24]:
```suggestion
3. You can set CASDOOR_HOSTNAME = `http://localhost:8000`. When deploy Casdoor in `prod` mode. See [production mode](https://casdoor.org/docs/basic/server-installation#production-mode).
```

**Review by leo220yuyaodog — CHANGES_REQUESTED** [review]

**Review by leo220yuyaodog — APPROVED** [review]

**Pull request merged**